src/coreModules/api/specEndpointFactory/index.js

```diff
@@ -0,0 +1,34 @@
+import openApi from 'dina-schema/build/openApi.json'
+
+const buildOperationIdPathnameMap = openApiSpec => {
```

Comment:
maybe it should be `paths` instead of depending on the shape of the `openApiSpec`

Reply:
its actually pathname since the query params are not included